modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml

```diff
@@ -50,14 +50,14 @@
     <Resource>
         <AuthType>Any</AuthType>
         <HttpVerb>GET</HttpVerb>
-        <Uri>http://localhost:9763/virtual_firealarm/manager/device</Uri>
-        <UriTemplate>/manager/device/*</UriTemplate>
+        <Uri>http://localhost:9763/virtual_firealarm/manager/device/{device_id}</Uri>
```

Comment:
why do we need to place localhost and 9763 port here? and this does not support https either. Cannot we simply keep the relative path here?